corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -393,16 +399,19 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
     if (!appId) {
         throw new Error('Attempt to refresh application for null appId');
     }
+    var sessionId = FormplayerFrontend.request('getCurrentSessionId');
```

Comment:
what happens when someone is on the main screen of app preview and hits update and sessionId is null?